backend/indexer/indexer.py

```diff
@@ -63,11 +67,15 @@ async def sync_data_source_to_collection(inputs: DataIngestionConfig):
     Returns:
         None
     """
-    client = await get_client()
-    await client.aupdate_data_ingestion_run_status(
+    loop = asyncio.new_event_loop()
+    def _run_async(coroutine):
+        return loop.run_until_complete(coroutine)
```

Comment:
We should not need a new loop, fastapi already runs an event loop

Reply:
Running the ingestion task in fastapi's event loop is the reason why it blocks the entire API